common.gypi

```diff
@@ -31,7 +31,7 @@
         'os_posix': 1,
         'v8_postmortem_support%': 'true',
       }],
-      ['GENERATOR == "ninja" or OS== "mac"', {
+      ['GENERATOR == "ninja" or OS=="mac"', {
```

Comment:
I was going by the fact that most other conditions referencing OS don't have any spaces so I just thought line 34 was meant to be the same. Initially I was going to edit the whole file so that every condition had spacing (since some don't and some do) but then I thought it was probably done this way for a reason, which made line 34 be inconsistent with either case.